cmd/manager/kodata/knative-eventing/2-upgrade-to-v0.14.2.yaml

```diff
@@ -15,6 +15,6 @@ spec:
       restartPolicy: Never
       containers:
       - name: upgrade-brokers
-        image: gcr.io/knative-releases/knative.dev/eventing/cmd/upgrade/v0.14.0@sha256:574b61b38c5bba600c6a00de39eb7d8b75bb38f551755c21eda938653b74bef3
+        image: gcr.io/knative-releases/knative.dev/eventing/cmd/upgrade/v0.14.0@sha256:8ad31868892f8730c818f23f45ea2fce278a37bc609954870f28f0bbfe3934ee
```

Comment:
Gotta change this once the image is published I didn't see any references to gcr.io other than this

Reply:
Couldn't find the correct image in https://github.com/openshift/release/blob/master/core-services/image-mirroring/knative/mapping_knative_v0_14_quay